api/src/backend/config/django/base.py

```diff
@@ -159,6 +159,27 @@
     {
         "NAME": "django.contrib.auth.password_validation.NumericPasswordValidator",
     },
+    {
+        "NAME": "api.validators.SpecialCharactersValidator",
+    },
```

Comment:
To maintain consistency, shouldn't we add something like "min_special": 1 here as well?

Reply:
Done in [`3fca219` (#8225)](https://github.com/prowler-cloud/prowler/pull/8225/commits/3fca219cd28f16efa70b89db034c7496f8860238)

Reply:
Thanks!!